code/drasil-website/lib/Drasil/Website/Example.hs

```diff
@@ -221,11 +234,19 @@ getCodePath path ex programLang = path ++ "code/stable/" ++ map toLower ex ++ "/
 -- | Uses 'exRt' path (srsDoxPath in this module).
 getDoxPath path ex programLang = path ++ map toLower ex ++ "/doxygen/" ++ programLang ++ "/index.html" -- need example path
 
+-- | Get the paths for the source code located in the code/drasil-example directory.
+getExampleSrcCodePath :: FilePath -> String -> FilePath
```

Comment:
Is there any chance you can change this change the type signature to `:: FilePath -> Example -> FilePath`? Alternatively, since we only use this function in `getSourceCodeRef`, we can just inline the function body. I don't think this is a very reusable function (since it should only be called once for each example and packaged in a chunk anyway).

Reply:
I think the latter option is better, actually.

Reply:
Moved it inline here:

https://github.com/JacquesCarette/Drasil/blob/ac9f9b53fcb6dc5c3185ca2753713ac3a4567d53/code/drasil-website/lib/Drasil/Website/Example.hs#L187-L198